cmd/arktos-network-controller/network-controller.go

```diff
@@ -47,6 +48,10 @@ func main() {
 		workers = defaultWorkers
 	}
 
+	if len(kubeAPIServerIP) == 0 {
```

Comment:
Nit: As KubeAPIserverIP is a must-have param, shall we change the check to net.ParseIP() to make sure the IP address is valid? Or even more, try whether the 6443 port in that address is reachable to make sure it is an APIServer?

Reply:
it makes sense to have a more stringent check; will update as suggested.